### PR TITLE
chore: drop Node.js v8 support

### LINKED ISSUE #5186: Drop Node.js 8 support
Per https://github.com/nodejs/Release#release-schedule, Node.js 8 reaches EOL at December 31st, 2019. We should then drop support for Node.js 8 in Puppeteer.

- [ ] Update CI configurations
- [ ] Leverage modern JS features in source code where applicable. Some features that Node.js v10 has that weren't in Node.js v8 are RegExp named capture groups, Unicode property escapes, async generators, `for`-`await`-`of`, one-sided string trimming, and optional catch binding.

### PR DESCRIPTION
**Pull request by hanselfmu**
Update Node.js versions to 10.18.1

Fixes #5186

### CODE REVIEW

File: experimental/puppeteer-firefox/.ci/node10/Dockerfile.windows [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+ENV NODE_VERSION 10.18.1
+
+RUN setx /m PATH "%PATH%;C:\nodejs"
+
+RUN powershell -Command \
+    netsh interface ipv4 set subinterface 18 mtu=1460 store=persistent ; \
+    Invoke-WebRequest $('https://nodejs.org/dist/v{0}/node-v{0}-win-x64.zip' -f $env:NODE_VERSION) -OutFile 'node.zip' -UseBasicParsing ; \
+    Expand-Archive node.zip -DestinationPath C:\ ; \
+    Rename-Item -Path $('C:\node-v{0}-win-x64' -f $env:NODE_VERSION) -NewName 'C:\nodejs'
```
Comment (mathiasbynens) [inline experimental/puppeteer-firefox/.ci/node10/Dockerfile.windows]: While we're here, let's also remove the `experimental/puppeteer-firefox/.ci/node{6,8}` folders.

**Review by mathiasbynens — APPROVED** [review]
LGTM, thanks!

**Comment by hanselfmu** [comment]
@googlebot I signed it!

File: experimental/puppeteer-firefox/.ci/node10/Dockerfile.windows (hunk 1 above)
Comment (hanselfmu) [reply experimental/puppeteer-firefox/.ci/node10/Dockerfile.windows]: They are removed now.

**Pull request merged**